types/sax-stream/sax-stream-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as saxStream from 'sax-stream';
```

Comment:
This file uses `import * as` to import either a class or a function. This is not a legal ES6 construct and this will be classified as an error in the future.

Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

If the module being typed is a CommonJS module (this is by far more common scenario, especially for NPM packages), then the correct form is
```ts
import x = require("y");
```

If the module being typed is an ES6 module with a *default export*, then the correct form is
```ts
import x from "y";
```

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Reply:
Should be resolved now.